specs/decorator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../decorator'
+
+describe Decorator do
+  let(:nameable_mock) { double('Nameable') }
+  subject { Decorator.new(nameable_mock) }
+
+  describe '#correct_name' do
+    it 'delegates correct_name to the wrapped Nameable object' do
+      expect(nameable_mock).to receive(:correct_name)
+      subject.correct_name
+    end
+  end
+end
```

Comment:
- [ ] @Team Great work adding tests for the decoration classes however you forgot to add tests for the  Nameable class, kindly consider adding test for it.

![image](https://github.com/nilab-khwaja/OOP-school-library/assets/61408860/15a2a8be-2050-4631-ba85-6dafb61bc61c)